src/storyMap/components/StoryMapsHomeCard.js

```diff
@@ -67,14 +66,16 @@ const StoryMapsHomeCard = props => {
 
   return (
     <HomeCard
-      aria-labelledby="story-maps-list-title"
-      sx={{
-        flexDirection: 'column',
+      id="story-maps"
+      isDefault={false}
+      title={t('storyMap.home_title')}
+      action={{
+        label: t('storyMap.home_create'),
+        to: 'tools/story-maps/new',
       }}
+      useImg={false}
```

Comment:
Issue: I don't think you need `useImg` you should be able to check if there it no `image` prop passed